libs/import/src/lib/view-extracted-elements/contract/utils.ts

```diff
@@ -183,41 +189,47 @@ export async function formatContract(
         return sellerId;
       }
     },
-    /* e */'term.territories': (value: string) => getStaticList('territories', value, separator, 'Territories', true, 'world') as Territory[],
-    /* f */'term.medias': (value: string) =>  getStaticList('medias', value, separator, 'Medias') as Media[],
-    /* g */'term.exclusive': (value: string) => {
+    /* e */'term[].territories_included': (value: string) => {
+      const result = getTerritoryList(value, separator) as Territory[];
+      return result;
```

Comment:
just return directly
```suggestion
      return getTerritoryList(value, separator) as Territory[];
```